packages/grid/helm/syft/templates/attestation/attestation-deployment.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.attestation.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: attestation
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+    app.kubernetes.io/component: attestation
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "common.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: attestation
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        {{- include "common.labels" . | nindent 8 }}
+        app.kubernetes.io/component: attestation
+    spec:
+      containers:
+        - name: attestation-container
+          image: {{ .Values.global.registry }}/openmined/grid-attestation:{{ .Values.global.version }}
```

Comment:
this image is not yet in the repo? is it a separate PR?

Reply:
ah i see there are two PRs:
- https://github.com/OpenMined/PySyft/pull/8643/files#diff-d58bb1b2d58992c665bb779ba6eddfc2e6ad56c9fe064b92d0f63f5f752dd882
- https://github.com/OpenMined/PySyft/pull/8659/files#diff-09319b2675ff5b9deb32aadf2fca875581cdbebadb62ed71f3bed02e95916b2e